<ui version="4.0" >
 <class>QUReportDialog</class>
 <widget class="QDialog" name="QUReportDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New Report</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images/uman.qrc" >
    <normaloff>:/control/report.png</normaloff>:/control/report.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="infoIconLbl" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="images/uman.qrc" >:/marks/information.png</pixmap>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="infoTextLbl" >
        <property name="text" >
         <string>...</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>5</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <widget class="QListWidget" name="reportList" >
         <property name="dragEnabled" >
          <bool>true</bool>
         </property>
         <property name="dragDropMode" >
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showBaseDirChk" >
         <property name="text" >
          <string>Show current songs path</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="spacing" >
        <number>5</number>
       </property>
       <item>
        <widget class="QToolButton" name="createPlainTextBtn" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Create 
Plain Text Report</string>
         </property>
         <property name="icon" >
          <iconset resource="images/uman.qrc" >
           <normaloff>:/types/text.png</normaloff>:/types/text.png</iconset>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="styleCombo" />
       </item>
       <item>
        <widget class="QToolButton" name="createHtmlBtn" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Create 
Html Report</string>
         </property>
         <property name="icon" >
          <iconset resource="images/uman.qrc" >
           <normaloff>:/types/page_code.png</normaloff>:/types/page_code.png</iconset>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="topMargin" >
      <number>5</number>
     </property>
     <property name="rightMargin" >
      <number>5</number>
     </property>
     <property name="bottomMargin" >
      <number>5</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>191</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="doneBtn" >
       <property name="text" >
        <string>Done</string>
       </property>
       <property name="icon" >
        <iconset resource="images/uman.qrc" >
         <normaloff>:/marks/accept.png</normaloff>:/marks/accept.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/uman.qrc" />
 </resources>
 <connections/>
</ui>
